src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.model.Movie;
+
+public interface MovieDao {
+    Movie add(Movie movie);
+
+    Optional<Movie> get(Long id);
```

Comment:
later we will ask to implement `getAll()` as well? it will be needed for controllers

Reply:
I guess yes. For now they don't know how to implement it